.gitignore

```diff
@@ -3,4 +3,4 @@ npm-debug.log
 /dist
 *.xml
 .idea/
-*.DS_Store
+*.DS_Store
```

Comment:
Can you undo this change? we want a newline at the end of the file

Reply:
fixed